checkstyle/suppressions.xml

```diff
@@ -272,6 +272,9 @@
     <suppress checks="MethodCount|NPathComplexity|MagicNumber" files="com/hazelcast/spi/Operation"/>
     <suppress checks="MethodCount" files="com/hazelcast/spi/impl/NodeEngineImpl"/>
     <suppress checks="ClassFanOutComplexity" files="com/hazelcast/spi/impl/NodeEngineImpl"/>
+    <!-- node engine constructs objects, so it knows many of them -->
+    <suppress checks="ClassDataAbstractionCoupling" files="com/hazelcast/spi/impl/NodeEngineImpl"/>
```

Comment:
I strongly suggest migrating away from suppressions.xml towards `@SuppressWarnings("checkstyle:classdataabstractioncoupling")`. It makes it much more visible that something was suppressed, plus it discourages wildcard suppressions.